packages/labs/gen-manifest/goldens/test-element-a/custom-elements.json

```diff
@@ -0,0 +1,40 @@
+{
+  "schemaVersion": "1.0.0",
+  "modules": [
+    {
+      "kind": "javascript-module",
+      "path": "element-a.js",
+      "summary": "TODO",
+      "description": "TODO",
+      "declarations": [
+        {
+          "kind": "variable",
```

Comment:
Can you add a variable that's of type `ElementA` to see if the type reference works?

Reply:
Done. Added a few more to test the various types of references.